front/app/components/ConsentManager/PreferencesModal/Preferences.test.tsx

```diff
@@ -12,7 +12,7 @@ describe('<Preferences />', () => {
 
   const categoryDestinations: CategorizedDestinations = {
     analytics: ['google_analytics', 'google_tag_manager'],
-    advertising: ['segment'],
+    advertising: ['google_analytics'],
```

Comment:
What's the reasoning behind this?

Reply:
It's just some mock data for the test. As we no longer have segment in the cookie policy, typescript complains so I just replaced it with another item.